### ISSUE TITLE
shipit_static_analysis: flake8-quotes should not be enabled

### ISSUE THREAD

**Issue by ahal** [mozilla-releng/services#777]
**MozReview URL:** https://reviewboard.mozilla.org/r/213006/#comment277984

**Problem:** The Q000 error comes from the 'flake8-quotes' plugin (which isn't used on central). The errors shown in the linked review can't be reproduced locally. There might be some default flake8 configuration lying around on the machine running reviewbot.

Though, we shouldn't be picking up configuration from outside of the tree either. So I'll look into how we can prevent this extra configuration from applying in general (though short term, it would be best to find and remove the flake8-quotes plugin).

**Comment by ahal** [comment]
The flake8 configuration handling is pretty bad. I think this will need to be fixed on shipit_static_analysis side for now, any in-tree fix is going to be fairly involved.

**Comment by sylvestre** [comment]
We were discussing about that on irc
```
$ ./mach lint testing/marionette/client/marionette_driver/geckoinstance.py
```
returns nothing.

**Comment by marco-c** [comment]
https://github.com/mozilla-releng/services/blob/master/src/shipit_static_analysis/requirements.nix#L346

**Comment by sylvestre** [comment]
It was enabled for releng service itself (and a bit overkill)
https://github.com/mozilla-releng/services/commit/c456619b3d64f40e4965a17cd8149752ac43c312
pr #389

I guess because we have the dependencies, flake8 uses it by default.

**Comment by ahal** [comment]
I do consider it a bug that |mach lint -l flake8| picks up external configuration, and filed:
https://bugzilla.mozilla.org/show_bug.cgi?id=1432331

Though if possible, it would be good to fix the issue here for now, as I don't see a clear path forward in the other bug.

**Comment by La0** [comment]
@ahal I made a PR for `shipit_static_analysis` to ignore flake8 `Q000` issues: #782. I'll update staging today, hopefully production by tommorow.

**Comment by ahal** [comment]
Awesome, thanks for the quick fix! Once bug 1432331 is fixed you guys can re-enable that plugin if you like (sorry for the inconvenience).
